common/src/main/resources/assets/wynntils/lang/de_de.json

```diff
@@ -1,47 +1,2655 @@
+  "command.wynntils.lootrun.noteRemovedSuccessfully": "Notiz erfolgreich entfernt bei %s mit dem Text \"%s\".",
+  "command.wynntils.lootrun.noteUnableToFind": "Notiz bei %s konnte nicht gefunden werden.",
+  "command.wynntils.lootrun.recordStart": "Aufzeichnung des Lootruns gestartet. Führen Sie %s aus, um die Aufzeichnung zu stoppen.",
+  "command.wynntils.lootrun.recordStop1": "Aufzeichnung des Lootruns gestoppt. Führen Sie %s aus, um den Lootrun zu löschen.",
+  "command.wynntils.lootrun.recordStop2": "Führen Sie %s aus, um den Lootrun zu speichern.",
+  "command.wynntils.lootrun.savedLootrun": "Lootrun erfolgreich gespeichert.",
+  "command.wynntils.lootrun.undoNotFarEnough": "Sie sind nicht weit genug entfernt, um die Bewegung rückgängig zu machen.",
+  "command.wynntils.lootrun.undoStandNear": "Stehen Sie in der Nähe eines vorherigen Ortes, um Bewegungen rückgängig zu machen.",
+  "command.wynntils.lootrun.undoSuccessful": "Bewegungen erfolgreich rückgängig gemacht.",
+  "command.wynntils.map.description": "Verwalten von kartenbezogenen Einstellungen",
+  "command.wynntils.onlineMembers.description": "Aktuelle Online-Spieler in einer Gilde anzeigen",
+  "command.wynntils.onlineMembers.lookingUp": "Online-Mitglieder werden gesucht...",
+  "command.wynntils.player.description": "Gildeninformationen und zuletzt gesehen für einen Spieler anzeigen",
+  "command.wynntils.player.lookingUp": "Spielerstatistiken werden gesucht...",
+  "command.wynntils.quest.description": "Quests auflisten, anzeigen und verfolgen",
+  "command.wynntils.reauth.tryReauth": "Verbindung zu Hades wird getrennt und erneute Authentifizierung wird versucht...",
+  "command.wynntils.reloadCaches.reloading": "Caches werden neu geladen...",
+  "command.wynntils.servers.description": "Informationen über Wynncraft-Server anzeigen",
+  "command.wynntils.statistics.clickHere": "Klicken Sie hier, um zu bestätigen",
+  "command.wynntils.statistics.description": "Statistiken anzeigen und zurücksetzen",
+  "command.wynntils.statistics.warnReset": "Dies wird alle gesammelten Statistiken für diesen Charakter zurücksetzen. Sind Sie sicher, dass Sie fortfahren möchten?",
+  "command.wynntils.territory.description": "Alias für /compass territory",
+  "command.wynntils.wynntils.description": "Bietet verschiedene Wynntils-Funktionen",
+  "core.wynntils.category.chat": "Chat",
+  "core.wynntils.category.combat": "Kampf",
+  "core.wynntils.category.commands": "Befehle",
+  "core.wynntils.category.debug": "Debuggen",
+  "core.wynntils.category.embellishments": "Verzierungen",
   "core.wynntils.category.inventory": "Inventar",
   "core.wynntils.category.map": "Karte",
   "core.wynntils.category.overlays": "Overlays",
   "core.wynntils.category.players": "Spieler",
-  "core.wynntils.category.redirects": "Umleitungen von Spielbenachrichtigungen",
-  "core.wynntils.category.tooltips": "Gegenstand Tooltips",
+  "core.wynntils.category.redirects": "Spielbenachrichtigungs-Weiterleitungen",
```

Comment:
I'm not sure GPT does this better than a human translator. Look at this example, for instance.